src/App.js

```diff
@@ -293,4 +295,6 @@ const AppWithRedux = connect(
 
 const AppWithReduxStore = () => <AppWithRedux store={store} />;
 
-AppRegistry.registerComponent('Rainbow', () => AppWithReduxStore);
+AppRegistry.registerComponent('Rainbow', () =>
+  ENABLE_DESIGN_SYSTEM_PLAYGROUND === 'true' ? Playground : AppWithReduxStore
```

Comment:
Can you move this to a component to make fast-refresh workable?

Reply:
Would that work when changing environment variables? I've been having to restart the app with a clean cache when changing the value of `ENABLE_DESIGN_SYSTEM_PLAYGROUND`, but I might be missing something.

Reply:
oh, you're absolutely right,
I think this might be unclear but I think this variable should be inside `src/config/defaultDebug.js`

We use this for all variables that are non-sensitive (so they don't need to keep in .env, can be reactive, don't require restarting packager, etc.). While building, this should make a copy of this file (`debug.js`) that will be a local file that you can change w/o changes in a repo (see: `scripts/postinstall.sh:102`) 



The other thing is that we're not very consequent in that and variables like `DEBUG_NM` or `DISABLE_REANIMATED_IN_DEV` should't be in `.env` (and I think I added both of them¯\_(ツ)_/¯).

Reply:
Oh cool, I didn't know that existed. I'll move it over.

Reply:
This is so much better. Fast refresh is working now too. Thanks for pointing this one out, it's a massive improvement 🙏